### PR TITLE
Translate block labels

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

**Comment by core23** [comment]
@rande @Soullivaneuh @sagikazarmark Translation please

File: Block/TimelineBlock.php [hunk 1]
```diff
@@ -99,11 +99,14 @@ public function buildEditForm(FormMapper $formMapper, BlockInterface $block)
             'keys' => array(
                 array('title', 'text', array(
                     'required' => false,
+                    'label'    => 'form.label_title',
                 )),
                 array('max_per_page', 'integer', array(
                     'required' => true,
+                    'label'    => 'form.label_per_page',
```
Comment (OskarStark) [inline Block/TimelineBlock.php]: should be ``form.label_max_per_page``

**Comment by rande** [comment]
form.label_title => titre
form.label_per_page => par page

**Comment by core23** [comment]
@OskarStark RTM

**Comment by OskarStark** [comment]
Thank you @core23

**Pull request merged**